index.ts

```diff
@@ -126,6 +154,8 @@ async function main() {
           if (projectIssue.state === "closed") continue;
           // if issue doesn't have the "Price" label then skip it, no need to pollute repo with draft issues
           if (!(projectIssue.labels as GitHubLabel[]).some((label) => label.name.includes(LABELS.PRICE))) continue;
+          if (projectIssue.assignee) continue;
```

Comment:
Shouldn't it be `projectIssue.assignee?.login`? I can't find ts types for the `assignee` object but, as far as I remember, it can be `null` hence `projectIssue.assignee?.login` is used everywhere in the code